Thorium-Sim/thorium-nova
# Test card

## Description

In development mode, adds a page for picking a card to test. It renders the card using the first player ship in the flight. Useful for developing cards.

## Related Issue








Refs #128

## How do you know the changes work correctly?






Manual Testing
- Start a flight.
- Click the "Cards" button in the top menu of the client lobby
- Choose on the cards.
- The station should render.

## Types of changes



      change)

## Checklist:
